### PR TITLE
Homework12part2

### PR DESCRIPTION
**Pull request by RomanPendyur**

### CODE REVIEW

File: HomeworkProject/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/indexStyles.css">    
+    <title>Document</title>
+</head>
+<body class="body__style">
```
Comment (roskyba) [inline HomeworkProject/index.html:10]: Poorly class name

File: HomeworkProject/styles/indexStyles.css [hunk 2]
```diff
@@ -0,0 +1,169 @@
+.body__style{
```
Comment (roskyba) [inline HomeworkProject/styles/indexStyles.css:1]: Do not forget to rename is CSS as well

File: HomeworkProject/styles/indexStyles.css [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+}
+
+.checkbox_input_form__style{
+    position: absolute;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+}
+
+.costom_checkbox_input_form__style{
```
Comment (roskyba) [inline HomeworkProject/styles/indexStyles.css:113]: `costom ===> custom`

**Comment by RomanPendyur** [comment]
Ok, I will try no repeat these errors.

**Pull request closed without merging**